api/resources/translations/messages-ne.properties

```diff
@@ -989,365 +1058,291 @@ related_entities.clinic.parent.parent.name = जिल्ला अस्पत
 task.state = कार्य अवस्था
 task.type = कार्य प्रकार
-task.warning.danger_sign = 
-task.warning.high_risk = 
-task.warning.home_birth = 
-tasks = पठाइएको संदेश
+tasks = पठाइएको सन्देश
 tasks.0.messages.0.message = सन्देश
 tasks.0.messages.0.to = पाउने
-tasks.0.state = राज्य
+tasks.0.state = अवस्था
 tasks.0.timestamp = समय/ मिति
-tasks.disabled = 
-tasks.none = 
-tasks.none.n.weeks = 
-testing.description = 
-testing.title = 
-title = 
+tasks.disabled = यस प्रयोगकर्ताका लागि कार्यहरू उपलब्ध गराइएको छैन।
+tasks.none = कार्यहरु छैनन्
+tasks.none.n.weeks = {WEEKS, plural, one{अर्को हप्तामा कुनै कार्यहरू छैनन्।} other{अर्को \# हप्तामा कुनै कार्यहरू छैनन्।}}
+tasks.group.leave = के तपाईं साँच्चै यो पृष्ठ छोड्न चाहनुहुन्छ? तपाईं यस घरपरिवारका अन्य कार्यहरू अब हेर्न सक्षम हुनुहुने छैन।
+tasks.group.completed = सबै घरपरिवारका कार्यहरू पूरा भए।
+tasks.group.title = अन्य घरपरिवारका कार्यहरू
+testing.description = हामीले लुकाएर राखेको परिक्षण पृष्ठ तपाईंले भेट्नु भएको छ। आफ्नो इन्स्टलेसन बिगार्न चाहन्नु हुन्न भने फिर्ता जानुहोस्। 
+testing.title = परिक्षण पृष्ठ
+title = शिर्षक
 to\ recipient = पाउने {{recipient}}
 today = आज
-tomorrow =
+tomorrow = भोलि
 training_cards.confirm.exit = यो तालिम सकिएको छैन । यदि तपाईंले अहिले छोड्नुभयो भने, तपाईंले आफ्नो प्रगति गुमाउनु हुनेछ र यसलाई पूरा गर्न पछि फेरि सम्झाइनेछ।
 training_cards.confirm.button.no = रद्द गर्नुहोस्
 training_cards.confirm.button.yes = छोड्नुहोस्
 training_cards.confirm.title = तालिम छोड्ने हो?
-translation.add =
-translation.key = 
-unique.id = 
-upgrade = 
-upgrade.description = 
-upload = 
-upload.favicon = 
-upload.header.logo = 
-upload.icon = अपलोड चिह्न
-upload.json.forms.help = फाइल फारम परिभाषा एक जेएसओएन इनकोडिंग सूची समावेश गर्नुपर्छ । कुनै पनि अवस्थित प्रकारका अधिलेखित गरिनेछ ।
-upload.json.forms.title = 
-upload.sms.forms = 
-upload.xform.context = 
-upload.xform.help = 
-upload.xform.meta.help = 
-upload.xform.title = 
-upload.xform.xml = 
-upload.xform.xml.help = 
-url = 
-user.fullname.help = 
-user.password.current = 
-user.phone.help = 
-user.place.help = 
-user.username.help = 
-username.invalid = 
-username.taken = 
-usertype._admin = 
+training_cards.error.loading = तालिम लोड गर्दा त्रुटी भयो। तपाइँको सुपरभाइजरलाई सम्पर्क गर्नुहोस्। 
+training_cards.error.save = तालिम सेभ गर्न त्रुटी।
+training_cards.form.saved = तालिम सम्पन्न भयो। 
+training_cards.modal.title = महत्वपुर्ण परिवर्तनहरु
+translation.add = नयाँ अनुवाद कुञ्जी थप्नुहोस्
+translation.key = अनुवाद कुञ्जी
+unique.id = आईडी
+unknown.contact = अपरिचिती सम्पर्क 
+upgrade = अपग्रेड
+upgrade.description = तपाईँको एपलाई कुनै रिलीज, बीटा, वा ब्रान्चमा अपग्रेड गर्न, पहिलो चरणको रूपमा स्टेज गर्न सिफारिस गरिन्छ। यसले प्रयोगकर्ताहरूलाई अवरोध नगरी पृष्ठभूमिमा इन्स्टलेसन तयारी गर्छ। स्टेजिंग पुरा भएपछि, अपग्रेडको लागि 'Install' क्लिक गर्नुहोस्। यो पछि, यसलाई पहिलाकै अवस्थामा  सकिँदैन, त्यसैले कृपया तपाईँको डेटा ब्याकअप गरिएको छ र तपाईँका प्रयोगकर्ताहरूलाई डाउनटाइमको जानकारी दिइएको छ भन्ने निश्चित गर्नुहोस्।
```

Comment:
```suggestion
upgrade.description = तपाईँको एपलाई कुनै रिलीज, बेटा, वा शाखामा अपग्रेड गर्न, पहिलो चरणको रूपमा स्टेज गर्न सिफारिस गरिन्छ। यसले प्रयोगकर्ताहरूलाई अवरोध नगरी पृष्ठभूमिमा इन्स्टलेसन तयारी गर्छ। स्टेजिंग पुरा भएपछि, अपग्रेडको लागि 'इन्स्टल गर्नुहोस्' मा क्लिक गर्नुहोस्। त्यस पछि, यसलाई पहिलाकै अवस्थामा लैजान सकिँदैन, त्यसैले कृपया तपाईँको डेटा ब्याकअप गरिएको छ र तपाईँका प्रयोगकर्ताहरूलाई सेवा अवरुद्ध रहने बारे जानकारी दिइएको छ भन्ने निश्चित गर्नुहोस्।
```

Reply:
शाखा doesn't give a good meaning here and it's not the right translation, so keeping branch as that makes it more meaningful.

Reply:
That is what you have added. I'm just trying to make it consistent:
https://github.com/medic/cht-core/blob/17b69b1bae970ae5bbacf2693b61d007a859485a/api/resources/translations/messages-ne.properties#L763